sites/layouts/documentation_pattern.html

```diff
@@ -0,0 +1,700 @@
+{%- else -%}
+<p>List of accessibility guidances to be followed to ensure component is used in respect of the accessibility guideline and EN 301 549 standard (including WCAG 2.1 Level AA).</p>
+{%- endif -%}
+
+<div class="wb-disable-allow" data-wb-json='{
+	"url": "#[api]",
+	"streamline": "true",
+	"mapping": [
+		{
+			"@type": "rdf:Alt",
+			"mapping": [
+				{
+					"template": "[data-yes]",
+					"test": "fn:isArray",
+					"value": "/a11yGuidance",
+					"mapping": [
+						{
+							"template": "template",
+							"value": "/",
+							"mapping": [
+								{ "selector": "li", "value": "/@value" }
+							]
+						}
+					]
+
+				},
+				{
+					"template": "[data-no]",
+					"test": "fn:isLiteral",
+					"value": "/a11yGuidance",
+					"mapping": null
+				}
+			]
+		}
+	]
+}'>
+	<template data-no>
+		<p>{% if lng == "fr" %}Il n’y a pas de directives d’accessibilité définies{% else %}There is no accessibility guidance defined{% endif %}</p>
+	</template>
+
+	<template data-yes>
+		<ul>
+			<template>
+				<li></li>
+			</template>
+		</ul>
+	</template>
+</div>
+
+<!--
+<p>Data ajax</p>
+<div>
+	<pre><code data-wb-ajax='{
+		"url": "samples/sample-1.html",
+		"type": "replace",
+		"encode": true
+	}'></code></pre>
+</div>-->
+
+<h2 id="variations">{% if lng == "fr" %}Variantes{% else %}Variations{% endif %}</h2>
+
+{%- if lng == "fr" -%}
+<p>Voici quelques exemples populaires. Il ne s’agit pas d’une liste exhaustive des variantes possibles. Configurez votre page en fonction du type de page, des besoins des utilisateurs et des conseils du système de conception.</p>
+{%- else -%}
+<p>Here are some popular examples. This isn’t an exhaustive list of possible variations. Configure your page based on page type, user needs and design system guidance.</p>
+{%- endif -%}
+
+<div class="wb-disable-allow" data-wb-json='{
+	"url": "#[api]/variations",
+	"mapping": [
+		{ "selector": "[data-name]", "value": "/name/{{lng}}" },
+		{ "selector": "[data-description]", "value": "/description/{{lng}}", "isHTML": true  },
+		{ "selector": "[data-guidance]", "value": "/guidance/{{lng}}", "attr": "href" },
+
+		{
+			"@type": "rdf:Alt",
+			"mapping": [
+				{
+					"template": "[data-status-stable]",
+					"test": "fn:getValue",
+					"assess": "/status",
+					"expect": "stable",
+					"mapping": null
+				},
+				{
+					"template": "[data-status-tobereviewed]",
+					"test": "fn:getValue",
+					"assess": "/status",
+					"expect": "to be reviewed",
+					"mapping": null
+				},
+				{
+					"template": "[data-status-provisional]",
+					"test": "fn:getValue",
+					"assess": "/status",
+					"expect": "provisional",
+					"mapping": null
+				},
+				{
+					"template": "[data-status-deprecated]",
+					"test": "fn:getValue",
+					"assess": "/status",
+					"expect": "deprecated",
+					"mapping": null
+				}
+			]
+		},
```

Comment:
There is no stability indicator because a design pattern out of scope of our versioning api

```suggestion
```